src/pages/extending-agent-app/index.mdx

```diff
@@ -16,8 +16,9 @@ Your app can extend several areas of the LiveChat Agent App or HelpDesk App inte
 
 - Chat Actions, by adding a new action button in the Chat Action menu (available only for LiveChat)
 - the Details section, by adding a new tab with your web content or a new widget in the default Customer Details tab
-- the Main menu, by adding a new, fullscreen section with your web content
+- the Fullscreen App, by adding a new, fullscreen section with your web content
```

Comment:
```suggestion
- the main menu, by adding a new, fullscreen section with your web content (Fullscreen App)
```

Reply:
maybe this way? This section is about locations and the location for fullscreen apps is the main menu

Reply:
Agree, that sounds better